MAUI/NavigationDrawer/Side-Pane-Content.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: Setting Sliding Panel Content in .NET MAUI NavigationDrawer | Syncfusion
+description: Learn here all about Setting Sliding Panel Content support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+
+
+# Setting Sliding Panel Content in .NET MAUI Navigation Drawer
+
+The drawer pane content is only viewable when the drawer is in the open state. Its content can be set as
+
+*	Header Content
+
+*	Drawer Content
+
+*	Footer Content
+
+N> Header and Footer content are optional but Drawer content is mandatory to allocate space for the drawer.
+		
+## Header Content
+
+As the name suggests it is displayed at the top of drawer. The `DrawerHeaderView` property is used to set the header content of drawer. 
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<navigationdrawer:SfNavigationDrawer x:Name="navigationDrawer">
+    <navigationdrawer:SfNavigationDrawer.DrawerSettings>
+        <navigationdrawer:DrawerSettings>
+            <navigationdrawer:DrawerSettings.DrawerHeaderView>
+                <Grid BackgroundColor="#1aa1d6">
+                    <VerticalStackLayout VerticalOptions="Center"
+                                         HorizontalOptions="Center">
+                        <Label Text="Header View"/>
+                    </VerticalStackLayout>
+                </Grid>
+            </navigationdrawer:DrawerSettings.DrawerHeaderView>
+        </navigationdrawer:DrawerSettings>
+    </navigationdrawer:SfNavigationDrawer.DrawerSettings>
+</navigationdrawer:SfNavigationDrawer>
+
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfNavigationDrawer navigationDrawer = new SfNavigationDrawer();
+DrawerSettings drawerSettings = new DrawerSettings()
+{
+    DrawerWidth = 250,
+};
+navigationDrawer.DrawerSettings = drawerSettings;
+this.Content = navigationDrawer;
+  
+{% endhighlight %}
+
+{% endtabs %}
+
+![Header]()
+
+## Header Height
+
+The height of the drawer header content can be adjusted using the `DrawerHeaderHeight` property.
+
+N> `DrawerHeaderView` will be disabled by setting the `DrawerHeaderHeight` to zero.
```

Comment:
N> The `DrawerHeaderView` can be disabled by setting the `DrawerHeaderHeight` to zero.

Reply:
Changed